app/assets/javascripts/koi/koi/datePicker.js

```diff
@@ -1,23 +1,23 @@
-! function ($)
-{
```

Comment:
I think this might be a false-positive in Github. The file no longer exists in the project.